### ISSUE TITLE
When a VBA facility name changes the section name needs to change

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#11628]
## Description
When a VBA facility name change comes in via migration from the facility API, The name on the node changes, but if it is a Regional Office, the corresponding section name does not update,  so the two become out of sync.

## Acceptance Criteria
- [ ] When a VBA regional office name changes, the corresponding section name should also change.
- [ ] Aliases for all  VBA facilities assigned to that regional office should also update to match.
- [ ] If there is a change to the manual process, it should be added to the issue template for a facility name change.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by JQuaggles** [comment]
@swirtSJW How often does this happen? Do we have an idea of how often they update the name? 

And will/could this potentially be resolved with any of the VBA work, or is this more specific to FL? 

ie I'm trying to understand the severity of the issue.

**Comment by kmariepat-cityfriends** [comment]
When you have a section intrinsically linked to a page title, there should be a runbook to update the section title, there is a redirect involved

**Comment by jilladams** [comment]
Do the updates to the [VBA facility name change runbook](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/.github/ISSUE_TEMPLATE/runbook-vba-facility-name-change.md) make this closeable?

**Comment by kmariepat-cityfriends** [comment]
We will revisit refining this after https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13272 is complete

**Comment by swirtSJW** [comment]
This was covered yesterday in refinement:  This issue is not related to #13272.  It is a current defect that needs to be resolved regardless of editors.  This should be handled automatically not by a runbook.

To Josh's earlier question of How often does it happen.  I am not sure.  We have not been tracking VBAs that long.  Mainly going by pattern with VAMCs and Vet Centers that seem to get renamed more often than we thought they should.

**Comment by swirtSJW** [comment]
There is also the potential that when a Regional office changes name, it will create a new section since it can not find an existing one that matches its own name.  This can be even more problematic than having them misnamed, because now its children will be assigned to the old section.

**Comment by jilladams** [comment]
Likely at risk for sprint boundary. (@kmariepat-cityfriends )

**Comment by jilladams** [comment]
From mid-sprint: 
- let's get this into the VBA runbook as a manual step
- Then: figure out automating. 

No concerns from POs re: sprint boundary.

**Comment by davidconlon** [comment]
not concerned about missing sprint boundary.

**Comment by davidmpickett** [comment]
Related tickets that may need to be updated based on implementation of this ticket (or vice versa)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/64934
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15381

**Comment by davidmpickett** [comment]
@mmiddaugh and I are meeting with CAIA on 10/11 to talk about VBA facilities naming schema. May make sense to hold this ticket until after that in case there are any decisions that could impact this. See: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15490

**Comment by davidmpickett** [comment]
**Prescott Vet Center** might be the right Vet Center to look at to understand some of the risks/possibilities here. Looks like the Vet Center is now split across 2 different Sections:

- [dr-cameron-mckinley-department-of-veterans-affairs-veterans-center](https://prod.cms.va.gov/section/vha/vet-centers/dr-cameron-mckinley-department-of-veterans-affairs-veterans-center)
  - Governs the Prescott Vet Center node
  - no users assigns to this section
- [prescott-vet-center
](https://prod.cms.va.gov/section/vha/vet-centers/prescott-vet-center)
  - Governs all other nodes related to Prescott (services, location list, outstation, etc.) as well as the users

## Screenshots

<details><summary> Prescott Vet Center Section governs everything _but_ Prescott Vet Center </summary>

![screencapture-prod-cms-va-gov-section-vha-vet-centers-prescott-vet-center-2023-10-06-13_37_28](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/5f9dec1b-8b2b-44f7-9f32-9bd275a1d01b)

</details>

<details><summary> Dr-Cameron-Mckinley Section governs just Prescott Vet Center </summary>

![screencapture-prod-cms-va-gov-admin-content-2023-10-06-13_46_51 (1)](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/fc723633-c349-4f82-819f-79987e0fdba6)

</details>

<details><summary> Users assigned to the two sections </summary>


![screencapture-staging-cms-va-gov-admin-people-2023-10-06-13_44_35](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/ec1700a0-9b84-4fe9-8319-5ed791966fb4)

![screencapture-staging-cms-va-gov-admin-people-2023-10-06-13_44_16](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/185577b2-81ee-4acc-81a0-98b7251c4457)


</details>

**Comment by davidmpickett** [comment]
Marking this as blocked by the Short Name proposal, because if there are multiple names for a VBA facility then this ticket would be dependent on which name is being used to created the Section

**Comment by xiongjaneg** [comment]
Jan. 4 UX sync, this is blocked on a spike to determine path forward for VBA offices "at" other locations

**Comment by davidmpickett** [comment]
Moving to National Rollout as it is blocked by a ticket that is postponed till national roll out

**Comment by davidmpickett** [comment]
Not sure if this is new information, or just a different lens on to the same issue, but using the [Facilities audit](https://prod.cms.va.gov/admin/content/facilities?title=&type_op=in&type%5B%5D=vba_facility&moderation_state=All&workbench_access_section__section=All) I found a bunch of "orphaned" VBA facilities that don't have a "Parent system or facility" listed and aren't Regional Offices

[vba_orphans.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/15080677/vba_orphans.xlsx)

**Comment by omahane** [comment]
Uncovered an issue while working on the Vet Center name change: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19101

**Comment by omahane** [comment]
> Not sure if this is new information, or just a different lens on to the same issue, but using the [Facilities audit](https://prod.cms.va.gov/admin/content/facilities?title=&type_op=in&type%5B%5D=vba_facility&moderation_state=All&workbench_access_section__section=All) I found a bunch of "orphaned" VBA facilities that don't have a "Parent system or facility" listed and aren't Regional Offices
> 
> [vba_orphans.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/15080677/vba_orphans.xlsx)

@mmiddaugh Can you confirm that these have parent system/facility in Sandy's DB?

**Comment by mmiddaugh** [comment]
@davidmpickett @omahane All facilities in the database have an associated `parent_office`

**Comment by Becapa** [comment]
@r-bartlett-gsa @davidmpickett The VBA facility migration is already set to use the "VBA Facilities" section as a default. This is the same thing that we are implementing for Vet Centers in #19204 . I'm not sure if this ticket is still needed.
